Docker build failing

Build Log:
```
##[error]The command '/bin/sh -c apt-get -y purge --auto-remove -o APT::AutoRemove::RecommendsImportant=false &&     apt-get -y purge --auto-remove python2-minimal &&     rm -rf /usr/lib/python2.7' returned a non-zero code: 100
```

It seems Python 2 package has finally been removed.

> E: Unable to locate package python2-minimal

We should remove our workaround from the Dockerfile.

--- a/Dockerfile
+++ b/Dockerfile
@@ -23,6 +23,4 @@ RUN npm run build --prefix /frontend_build && \
     mv /frontend_build/build/* /app/static/ && \
     rm -rf /frontend_build
 
-RUN apt-get -y purge --auto-remove -o APT::AutoRemove::RecommendsImportant=false && \
-    apt-get -y purge --auto-remove python2-minimal && \
-    rm -rf /usr/lib/python2.7
+RUN apt-get -y purge --auto-remove -o APT::AutoRemove::RecommendsImportant=false

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.9-slim
 
 ENV PIP_NO_CACHE_DIR=1
 

